vestapol/__init__.py

```diff
@@ -1,9 +1,8 @@
-from vestapol import web_resources
-from vestapol import destinations
```

Comment:
tbh i only made the change to make flake8 happy. personally i would lean toward the more redundant method just because it is more transparent (don't have to look at the init files to figure out where things are coming from). We could look into auto-generating some sphinx docs to make this info easier to find.